src/services/hotel.service.js

```diff
@@ -0,0 +1,56 @@
+import db from '../models';
+
+/**
+ * Hotel service
+ */
+class HotelService {
+  /**
+     * Creates a Hotel
+     * @param {Object} params Information about hotel
```

Comment:
These descriptions are not directly matching with the `params` parameter in the method. And some IDEs may complain about that. Maybe you need something like:
```
  /**
     * Creates a Hotel
     * @param {Object} params Information about hotel
     *                 params.locationId locationId for the hotel
     *                 params.name name of the hotel
     *                 params.street street for the hotel
     *                 params.description description for the hotel
     *                 params.services services offered by the hotel
     * @returns {Object} hotel
     */
```

Reply:
thanks for feedback @gildniy, I would use that but I think that it is missing `datatypes of object properties`

I used this resource for documenting `Parameters with properties`.
https://jsdoc.app/tags-param.html